### ISSUE TITLE
Storm32 Gimbal and ArduCopter 4.3

### ISSUE THREAD

**Issue by RJCod73** [ArduPilot/ardupilot#24238]
**Issue details**
I am having a problem controlling my Storm32 gimbal using the software payload controls (Mission Planner and QGroundcontrol) 
I have tried the Mavlink config and the serial config, followed OlliW and Ardupilot documentation to the letter multiple times, cannot find a resolve for my issue in the forums, I would really appreciate any help on this. The strange thing is, someone was testing something using this autopilot while I was out of town and somehow changed the ardupilot FW to a VTOL TILTROTOR config and I didn't realize it until after I got the gimbal working so when I wanted to go flight test the gimbal I noticed this, changed it to the Quad-X config through Mission Planner and now it doesn't show any love no matter what I do to the parameters.

**Configuration**
Gimbal: Storm32 board v1.3, FW v.90, 2 axis (Pitch and roll/tilt and roll)
UAV: Quad-X on ArduCopter v4.3.7, mRo Control Zero H7 AP, 

[ParamFiles.zip](https://github.com/ArduPilot/ardupilot/files/11970006/ParamFiles.zip)

**Comment by amilcarlucas** [comment]
For support questions, use the forum. github is only for reproduceble bugs.

**Comment by rmackay9** [comment]
@RJCod73,

Yes, sorry to be strict but if we're not then our to-do list turns into another support forum.  Here's the place to post https://discuss.ardupilot.org/c/arducopter/copter43/167

**Comment by RJCod73** [comment]
Thank you, I appreciate you being direct and helpful. @rmackay9 you are exactly the person I was hoping to get in contact with, I made my profile on ardupilot.org but my profile and topic are on hold but I did s you asked and attached my most recent logs from bench top testing yesterday to make it easier for you. I am new to all of this and again I appreciate both of your direct info and quick response.
